### PR TITLE
Lint everything else

### PR DESCRIPTION
**Pull request by fregante**
Manual lint of the last few rules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,18 +47,9 @@
     ],
     "prettier": true,
     "rules": {
-      "camelcase": "off",
       "ava/no-import-test-files": "off",
```
Comment (fregante) [inline package.json:50]: Until https://github.com/avajs/eslint-plugin-ava/issues/311

File: package.json [hunk 2]
```diff
@@ -47,18 +47,9 @@
...
       "ava/no-skip-test": "off",
-      "ava/no-statement-after-end": "off",
-      "eqeqeq": "off",
-      "no-useless-call": "off",
-      "no-var": "off",
-      "prefer-rest-params": "off",
-      "prefer-spread": "off",
-      "unicorn/explicit-length-check": "off",
-      "unicorn/prevent-abbreviations": "off"
+      "camelcase": "off"
```
Comment (fregante) [inline package.json:51]: Due to GitHub’s API

**Review by fregante — COMMENTED** [review]
There was a test being skipped, but it seems to work. Feel free to revert the last commit if the test isn't ready.

File: test/simple.js [hunk 3]
```diff
@@ -73,41 +73,43 @@ test.cb('should not delete a fork that has more branches', (t) => {
 	});
 });
 
-// Currently our logic does not delete this fork
-test.skip('should delete a fork that has more branches, but all at upstream branch tips', (t) => {
```
Comment (denis-sokolov) [inline test/simple.js:77]: Hecks knows, I don’t remember why it works. Perhaps I had added the functionality afterwards and forgot to enable the test? Weird.

File: test/simple.js [hunk 4]
```diff
@@ -73,41 +73,43 @@ test.cb('should not delete a fork that has more branches', (t) => {
...
-	const mock = lib.mock({
-		listBranches(arguments_) {
-			if (arguments_.user === lib.USER.login) {
+test.cb(
+	'should delete a fork that has more branches, but all at upstream branch tips',
```
Comment (denis-sokolov) [inline test/simple.js:77]: eslint strangely formats this. If the `test.cb` part is anything other than `test.skip`, it puts arguments on separate lines. If it’s `test.skip`, then both arguments are on one line. That’s with ava/no-skip-test disabled.

At first I thought it’s due to the line length, but no, even in the PR diff the shorter version is wrapped, and the longer isn’t. And, also, `test.ski` is wrapped, and `test.skipp` is wrapped. Weird.

**Review by denis-sokolov — APPROVED** [review]
Thanks!

File: test/simple.js (hunk 4 above)
Comment (fregante) [reply test/simple.js:77]: That’s Prettier. Yeah I don’t like Prettier’s wrapping either 😉 
XO doesn’t care about line length as long as there’s no more than one `;` per line.

Comment (denis-sokolov) [reply test/simple.js:77]: You’re right that it’s prettier! https://github.com/prettier/prettier/issues/9285

**Pull request merged**